File: character-generator-starter/src/main/java/de/hipp/pnp/CharacterGeneratorApplication.java
package de.hipp.pnp;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication(scanBasePackages = "de.hipp.*")
public class CharacterGeneratorApplication {

	public static void main(String[] args) {
		SpringApplication.run(CharacterGeneratorApplication.class, args);
	}
}

File: character-generator-starter/src/main/kotlin/de/hipp/pnp/CharacterGeneratorApplication.kt
package de.hipp.pnp

import io.github.oshai.kotlinlogging.KotlinLogging
import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication

private val logger = KotlinLogging.logger {}

@SpringBootApplication
class CharacterGeneratorApplication {
    companion object {
        @JvmStatic
        fun main(args: Array<String>) {
            SpringApplication.run(CharacterGeneratorApplication::class.java, *args)
        }
    }
}
